src/components/mgt-person/mgt-person.ts

```diff
@@ -333,6 +337,23 @@ export class MgtPerson extends MgtTemplatedComponent {
     this.requestUpdate();
   }
 
+  /**
+   * async promise, returns Graph contact and/or Person associated with the email provided
```

Comment:
Should this move to mgt-person.graph or the other graph files?

Reply:
Good idea :)